package.json

```
@@ -4,7 +4,10 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "nodemon --exec babel-node src/index.js ",
```

Please remove the extra space i.e. 
```suggestion
    "dev": "nodemon --exec babel-node src/index.js",
```

---

I have fixed it